pyproject.toml

```diff
@@ -90,70 +89,3 @@ max-line-length=120
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
-
-[tool.pylint.master]
```

Comment:
Intentional to remove all of the below?

Reply:
There were duplicate keys.  If you recheck the file, it should all be in there.  That is why a previous commit or two failed.  Git really let me down today.